src/metrics.jl

```diff
@@ -289,6 +293,10 @@ end
 end
 rogerstanimoto{T <: Bool}(a::AbstractArray{T}, b::AbstractArray{T}) = evaluate(RogersTanimoto(), a, b)
 
+
+# Utility
+sqr(x) = x*x
```

Comment:
I swear I've seen cases (recently) where `x^2` calls `Base.power_by_squaring` (which is a performance disaster), but I'm having trouble coming up with a concrete example. I can change it.